app/scripts/lib/ppom/security-alerts-api.ts

```diff
@@ -20,6 +22,10 @@ export async function validateWithSecurityAlertsAPI(
   return postRequest(endpoint, request);
 }
 
+export async function getSupportedChains(): Promise<Hex[]> {
```

Comment:
Same suggestions as the mobile PR 😄

Reply:
All suggestions applied the same as in mobile. https://github.com/MetaMask/metamask-extension/pull/25716/commits/36c4290737866c6255972bdc6b8e147b4ffacba0